Index: src/sample/Test.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/sample/Test.java	(revision 6cab168227066851273313fe7612786f2973fb03)
+++ src/sample/Test.java	(date 1539891422748)
@@ -10,7 +10,6 @@
         Tesseract tesseract = new Tesseract();
         try {
             tesseract.setDatapath("tessdata");
-
             String text = tesseract.doOCR(new File("C:/Users/Pawel/Desktop/Projekt Inż/SerialDetector/Adaptivemean_thresh_binary.jpg"));
             System.out.print(text);
         } catch (TesseractException e) {
